### PR TITLE
StyleGAN-NADA Train and Inference Notebook

### PR DESCRIPTION
**Pull request by soumik12345**

### CODE REVIEW

**Comment by tcapelle** [comment]
Tell me when you are ready of a review =)

**Comment by soumik12345** [comment]
> Tell me when you are ready of a review =)

I think the notebook's ready for a review

File: colabs/stylegan_nada/StyleGAN-NADA.ipynb [hunk 1]
```diff
@@ -0,0 +1,658 @@
+{
```
Comment (tcapelle) [inline colabs/stylegan_nada/StyleGAN-NADA.ipynb:1]: <p>Maybe splitting in more cells. So if people re run the notebooks don't need to wget and re install everything</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (tcapelle) [inline colabs/stylegan_nada/StyleGAN-NADA.ipynb:1]: <p>also split here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

**Pull request merged**